chain-api/src/types/LaunchPadSale.ts

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright (c) Gala Games Inc. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  BigNumberProperty,
+  ChainKey,
+  ChainObject,
+  StringEnumProperty,
+  TokenInstanceKey
+} from "@gala-chain/api";
+import BigNumber from "bignumber.js";
+import { Exclude, Type } from "class-transformer";
+import { IsNotEmpty, IsString } from "class-validator";
+import { JSONSchema } from "class-validator-jsonschema";
+
+export enum SaleStatus {
+  ONGOING = "Ongoing",
+  END = "Finished"
+}
+
+@JSONSchema({
+  description: "Represents a launchpad sale, defining token quantities, pricing, and sale operations."
+})
+export class LaunchPadSale extends ChainObject {
+  @Exclude()
+  static INDEX_KEY = "LAUNCHPAD";
```

Comment:
```suggestion
  static INDEX_KEY = "GCLPS";
```
As @sentientforest mentioned in the last review, these should follow our convention GC<acronym>.